app/app.go

```diff
@@ -249,3 +253,17 @@ func (a *app) RemoteTerminalAllowed(
 	}
 	return true, nil
 }
+
+func (a *app) GetSessionRecording(ctx context.Context, id string, w io.Writer) (err error) {
+	err = a.store.GetSessionRecording(ctx, id, w)
+	return err
+}
+
+func (a *app) SaveSessionRecording(ctx context.Context, id string, sessionBytes []byte) error {
+	err := a.store.SetSessionRecording(ctx, id, sessionBytes)
+	return err
+}
+
+func (a app) GetStore() store.DataStore {
+	return a.store
+}
```

Comment:
Can we replace this with `GetRecorder`? I don't like passing `DataStore` to the api layer.

Reply:
sure! fixed.